### ISSUE TITLE
Uncaught TypeError: o.toArray is not a function

### ISSUE THREAD

**Issue by moabtools** [grpc/grpc-web#457]
Hi!

I try to work with any.proto

```
syntax = "proto3";
import "google/protobuf/any.proto";
...
message AddJobRequest {
	string api_key = 1;
	int32 token_id = 2;
	JobAction action = 3;
	google.protobuf.Any payload = 4;
}
```

I compile proto under Windows with command
```
protoc --plugin=protoc-gen-grpc-web=protoc-gen-grpc-web-1.0.3-windows-x86_64.exe --grpc-web_out=out=Job_grpc_web_pb.js,import_style=commonjs,mode=grpcweb:JS_Campaigns Job.proto
```
and pack with
```
npx webpack client.js
```

My function is:
```
function addNewJob(api_key, token_id, action, payload, callback) {
	
	var request = new AddJobRequest();
	request.setApiKey(api_key);
	request.setTokenId(token_id);
	request.setAction(action);
	request.setPayload({ payload: payload });
	
	job_client.addNew(request, metadata, (err, response) => {
		callback(response, err);
	});	
	
}
```

I receive error
```
Uncaught TypeError: o.toArray is not a function
```

The same error occurs with google/protobuf/wrappers.proto
What I'm doing wrong?

**Comment by mfecteau34** [comment]
I have the same error with google/protobuf/struct.proto.  I use linux with the last published versions of grpc-web (1.0.3) and protoc (3.6.1).

I hope it will be fixed. Otherwise, I do no see any workaround except to go back to REST ...

**Comment by mfecteau34** [comment]
It's a bug in the wrapRepeatedField function of jspb (google-protobuf).

It's the same error as [here](https://github.com/protocolbuffers/protobuf/issues/5162).

That's a serious bug that seems to affect many people.  Unfortunately, it prevents me from using grpc-web.

**Comment by mfecteau34** [comment]
After all, I think it was not a bug.

Try this :

```
function addNewJob(api_key, token_id, action, payload, callback) {
	
	var request = new AddJobRequest();
	request.setApiKey(api_key);
	request.setTokenId(token_id);
	request.setAction(action);
    
        var payloadAny = new proto.google.protobuf.Any({ payload: payload });
	request.setPayload(payloadAny);
	
	job_client.addNew(request, metadata, (err, response) => {
		callback(response, err);
	});		
}
```

**Comment by moabtools** [comment]
It's not working out of box
``` Uncaught TypeError: Cannot read property 'google' of undefined ```
Maybe I need to require something in my commonjs?

**Comment by mfecteau34** [comment]
It's difficult to tell why you have "proto" as undefined without looking at all your "client.js" script.  For my part, just by following the basic ["helloworld" example](https://github.com/grpc/grpc-web/blob/master/net/grpc/gateway/examples/echo/commonjs-example/client.js), I got this "proto" variable in my globals.

If it could help you : 
I was able to get some inspiration by looking at [this example](https://medium.com/@waynelpu/how-to-pass-plain-javascript-object-with-gprc-94a91906ab1c).  Though I didn't follow the example step by step, it was by looking at it that I discoved that there was a "fromJavaScript" function (and also a "toJavaScript") convert JSON to Struct.proto (and  vice-versa).
